__tests__/app.test.js

```diff
@@ -115,17 +116,76 @@ describe("GET /api/articles", () => {
         });
       });
   });
-  it("Sorts the array by date in descending order", () => {
+
+  it("Sorts the array by date in descending order by default", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
       .then(({ body: { articles } }) => {
         expect(articles).toBeSortedBy("created_at", {
-          coearce: true,
+          coerce: true,
           descending: true,
         });
       });
   });
+
+  it("Sorts the array by the specified column in the specified order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=title&order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toBeSortedBy("title", {
+          coerce: true,
+          descending: false,
+        });
+      });
+  });
+
+  const sort_by = [
+    "author",
+    "title",
+    "article_id",
+    "topic",
+    "created_at",
+    "votes",
+    "article_img_url",
+  ];
+  const order = ["asc", "desc", "ASC", "DESC"];
+
+  sort_by.forEach((sortField) => {
+    order.forEach((orderType) => {
+      it(`Sorts the array by ${sortField} in ${orderType} order regardless of case`, () => {
```

Comment:
I have very mixed feelings about this approach. Using loops to test multiple test cases isn't something I am sure is consistent with integration testing 🤔 
In integration testing I wouldn't say it is necessary to test every possible combination of possible `sort` and `order` queries like this, we could derive confidence from testing with less cases because it's bringing together multiple functions